src/spv/bitcoin/BRWallet.cpp

```diff
@@ -875,7 +1020,14 @@ BRTransaction *BRWalletCreateTxForOutputs(BRWallet *wallet, const BRTxOutput out
         balance += tx->outputs[o->n].amount;
 
         // fee amount after adding a change output
-        feeAmount = _txFee(wallet->feePerKb, BRTransactionVSize(transaction) + TX_OUTPUT_SIZE);
+        if (feeRate > 1000)
+        {
+            feeAmount = _txFee(feeRate, BRTransactionVSize(transaction) + TX_OUTPUT_SIZE);
+        }
+        else
+        {
+            feeAmount = _txFee(wallet->feePerKb, BRTransactionVSize(transaction) + TX_OUTPUT_SIZE);
+        }
```

Comment:
Now feeRate here is adjusted in line 963

Reply:
https://github.com/DeFiCh/ain/pull/308/commits/a6aae235b22b244bd371a191bb58f0d12c8faab7